client/omnichannel/directory/ContactForm.js

```diff
@@ -146,18 +146,18 @@ export function ContactNewEdit({ id, data, reload, close }) {
 
 		const payload = {
 			name,
-			email,
-			phone,
 		};
+		payload.phone = phone;
+		payload.email = email;
+		payload.livechatData = livechatData;
```

Comment:
We should call this property as `customFields` instead of `livechatData`. The first option is straightforward.